prow/jobs/test-infra/test-infra.jobs.yaml

```diff
@@ -0,0 +1,188 @@
+presubmits: # runs on PRs
+  kyma-project/test-infra:
+    - name: prow/test-infra/bootstrap
+      run_if_changed: "images/bootstrap/"
+      context: prow/test-infra/bootstrap
+      optional: true
+      skip_report: true # from documentation: SkipReport skips commenting and setting status on GitHub.
+      max_concurrency: 10
+      labels:
+        preset-dind-enabled: "true"
+        preset-sa-gcr-push: "true"
+        preset-docker-push-repository: "true"
+        preset-build-pr: "true"
+      decorate: true
+      path_alias: github.com/kyma-project/test-infra
+      spec:
+        containers:
+          - image: eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
```

Comment:
my only concern is that we are using bootstrap:0.0.1 to build bootstrap image itself. I know we have "chicken&egg" problem here but maybe there's a safe solution for that.

Reply:
This will be discussed later this week. Let's merge this PR as it is a working solution. If we come up with a better solution, I'll create another PR.